test_cases.cpp

```diff
@@ -0,0 +1,84 @@
+#include "uthread.h"
+#include <iostream>
+#include <unistd.h>
+
```

Comment:
@gruid018 the test case structure looks ok right?
we'll add different functions and then have different versions of worker functions and then, in the end, can give a menu maybe

Reply:
yeah sounds good to me!